channels/channels.go

```
@@ -135,7 +76,7 @@ func (VirtualConnections VirtualConnections) HandleOut(w http.ResponseWriter, r
 	w.(http.Flusher).Flush()
 
 	// Wait for connection established
-	connection.ServerConn = <-connection.Connected
+	<-connection.SambaConnected
```

In theory it should not be needed, current approach will fail faster because of channel closing on errors:
https://github.com/zentyal/gorpcproxy/pull/13/files#diff-03663b7fb16baf7927d5d34c6a0deec1R225
https://github.com/zentyal/gorpcproxy/pull/13/files#diff-03663b7fb16baf7927d5d34c6a0deec1R232